app/views/ovens/show.html.haml

```diff
@@ -3,7 +3,7 @@
 .cookie-info
   Cookie in oven:
   - if @oven.cookie
-    1 cookie with #{@oven.cookie.fillings || "no fillings"}
+    1 cookie with #{@oven.cookie.fillings.present? || "no fillings"}
```

Comment:
In Ruby, except for false and nil which are always false, everything else is true, for example here the string empty, with which the fillings attribute is initialized